fastmri/data/subsample.py

```diff
@@ -48,7 +48,13 @@ class MaskFunc:
     and ``EquispacedMaskFunc``.
     """
 
-    def __init__(self, center_fractions: Sequence[float], accelerations: Sequence[int]):
+    def __init__(
```

Comment:
As a side note, the `MaskFunc` is not a great name (not very informative term on what this class is used for)

Reply:
I'll consider changing it in the future - for now I'd like to keep for backwards compatibility.